### ISSUE TITLE
Rate limit exceeded: 413

### ISSUE THREAD

**Issue by timappelmansvlaanderen** [bbernhard/signal-cli-rest-api#374]
### The problem

When calling the register for a number, it works fine for different number, however it fails for one with the following message:

"error": "Failed to register: [413] Rate limit exceeded: 413\n"

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

Normal Mode

### What's the architecture of your host system?

x86-64

**Comment by bbernhard** [comment]
Please enable the debug mode and run the `signal-cli` command directly with `--verbose` for additional debug output (see https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/DEBUG.md for details)

**Comment by timappelmansvlaanderen** [comment]
It worked after 24 hours, got a 201 return value, but now adding a member to a group, i get that the number is nog registered, this used to work before

**Comment by bbernhard** [comment]
Are you still using the `normal` mode? Have you tried calling the `receive` endpoint? (In `normal` /`native` mode, you need to call the `receive` endpoint regularily)

**Comment by timappelmansvlaanderen** [comment]
yes, we call the receive regularly, we get a 201 created back from the "/v1/register/", then we call the API to add this member to the group, and we get the error:     "error": "WARN  RecipientHelper - CDSI request failed, trying fallback to CDS\njava.io.IOException: java.net.ProtocolException: Expected HTTP 101 response but was '404 Not Found'\n\tat org.whispersystems.signalservice.api.SignalServiceAccountManager.getRegisteredUsersWithCdsi(SignalServiceAccountManager.java:579)\n\tat org.asamk.signal.manager.helper.RecipientHelper.getRegisteredUsersV2(RecipientHelper.java:164)\n\tat org.asamk.signal.manager.helper.RecipientHelper.getRegisteredUsers(RecipientHelper.java:113)\n\tat org.asamk.signal.manager.helper.RecipientHelper.getRegisteredUser(RecipientHelper.java:129)\n\tat org.asamk.signal.manager.helper.RecipientHelper.lambda$resolveRecipient$0(RecipientHelper.java:91)\n\tat org.asamk.signal.manager.storage.recipients.RecipientStore.resolveRecipient(RecipientStore.java:206)\n\tat org.asamk.signal.manager.helper.RecipientHelper.resolveRecipient(RecipientHelper.java:89)\n\tat org.asamk.signal.manager.helper.RecipientHelper.resolveRecipients(RecipientHelper.java:78)\n\tat org.asamk.signal.manager.ManagerImpl.updateGroup(ManagerImpl.java:415)\n\tat org.asamk.signal.commands.UpdateGroupCommand.handleCommand(UpdateGroupCommand.java:141)\n\tat org.asamk.signal.App.handleLocalCommand(App.java:289)\n\tat org.asamk.signal.App.init(App.java:229)\n\tat org.asamk.signal.Main.main(Main.java:60)\nCaused by: java.net.ProtocolException: Expected HTTP 101 response but was '404 Not Found'\n\tat okhttp3.internal.ws.RealWebSocket.checkUpgradeSuccess$okhttp(RealWebSocket.kt:224)\n\tat okhttp3.internal.ws.RealWebSocket$connect$1.onResponse(RealWebSocket.kt:170)\n\tat okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\nThe user +32475828322 is not registered.\n"

**Comment by bbernhard** [comment]
Please try with the latest version and create another issue in case the problem still exists.
